en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,470 @@
+# Get Started with Choreo
```

Comment:
Regarding the flow diagram, the end users don't create accounts in Asgardeo do they? They create it on Choreo?

Reply:
They create the users in Asgardeo